glossary.yml

```
@@ -5180,18 +5270,18 @@
   en:
     term: "tibble"
     def: >
-      A modern replacement for R's data frame, which stores tabular data in columns
+      A modern replacement for [R's](#r_language) data frame, which stores tabular data in columns
```

Consistent with earlier examples of this

```suggestion
      A modern replacement for [R](#r_language)'s data frame, which stores tabular data in columns
```